tests/test_change_speed.py

```diff
@@ -4,12 +4,39 @@
 
 input_signal = np.genfromtxt('tests/data/bark.csv', dtype = 'float32')
 
+'''
+Test that exception is raised for 0 as a rate argument.
+'''
+def cutoff_zero():
```

Comment:
Same here, has to start with `test_`.

Reply:
Also the name `cutoff_zero` only applies to the low pass filter function. This is for the rate of change.